src/main/java/WriteFile.java

```diff
@@ -0,0 +1,58 @@
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * This class writes a txt file to a predetermined path
+ * Referenced from topic W3.3c
+ */
+public class WriteFile {
+
+    public static final String ADDRESS = "data/duke.txt";
+
+    public WriteFile(ArrayList<Task> store) throws IOException {
```

Comment:
Store might not be the best name for a task list, maybe naming it taskList might be a better choice?